debian/clickhouse-server.install

```diff
@@ -2,5 +2,6 @@ usr/bin/clickhouse-server
 usr/bin/clickhouse-copier
 usr/bin/clickhouse-report
 etc/clickhouse-server/config.xml
+etc/clickhouse-server/config.d/*.xml
```

Comment:
Why you don't add new configuration to default config.xml? We try not to bring anything to production via  `config.d` and users can use `config.d` safely without conflicts with our packages